src/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -296,10 +301,10 @@
           context (if updater
                     (assoc context :updater updater)
                     context)
-          _       (let [authorized? (if-let [wl (:certificate-whitelist puppetdb)]
-                                      (build-whitelist-authorizer wl)
-                                      (constantly true))
-                        app (server/build-app :globals globals :authorized? authorized?)]
+          _       (let [authorizer (if-let [wl (:certificate-whitelist puppetdb)]
```

Comment:
Like I suggested in my comment below, this could really just be ` app (server/build-app :globals globals :authorizer (when-let [wl (:certificate-whitelist puppetdb)] (build-whitelist-authorizer wl)))` since we're defaulting at a lower level already.